src/resolvers/profileResolver.ts

```diff
@@ -0,0 +1,54 @@
+import { Profile } from '../models/user'
+import cloudinary from 'cloudinary';
+import path from 'path';
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET
+});
+
+const profileResolvers = {
+    Query: {
+        getProfile: (parent:any , args: any) => {
+            return Profile.findById(args.id)
+        }
+    },
+    Mutation: {
+        updateProfile: async (parent:any, args:any, context: any) => {
+            try {
+              const { id, firstName, lastName, address, city, country, phoneNumber, biography, fileName } = args
+
+              const { userId } = context;
+
+                if (!userId) {
+                    throw new Error("You need to login first")
+                }
+
+                if (fileName)  {
```

Comment:
The variable that is defined here will be available for the following block

Referring to the `photo` which you have also referenced below, which will raise an error of `undefined` due to scoped nature of constant